opm/output/OutputWriter.hpp

```diff
@@ -74,44 +75,28 @@ class OutputWriter {
      * This routine should be called before the first timestep (i.e. when
      * timer.currentStepNum () == 0)
      */
-    virtual void writeInit(const SimulatorTimerInterface &timer, const NNC& nnc = NNC()) = 0;
+    virtual void writeInit( const NNC& nnc = NNC() ) = 0;
```

Comment:
Is that necessary?  The fighting with NNC has not been caused by this.  I furthermore think that a default NNC here makes sense, since if we don't have NNC information to write to output, it shouldn't be up to the caller of this method to create an empty NNC object.